relayer-cli/tests/integration.rs

```diff
@@ -105,12 +105,15 @@ fn query_channel_id() {
 fn query_client_id() {
```

Comment:
It would be nice to have a test/ simapp with two connections

Reply:
I believe in order to have two connections some configuration should be changed in the simapp. I looked into that and couldn't figure out where to change it maybe need @greg-szabo help. I see that in /simapp/config/genesis.json the connections are configured there but not sure how is the configuration passed as an argument to relayer (-c relayer/test/config/fixtures/relayer_conf_example.toml) also has [[connections]] so not sure how these two play out.

Reply:
> not sure how is the configuration passed as an argument to relayer (-c relayer/test/config/fixtures/relayer_conf_example.toml) also has [[connections]] so not sure how these two play out.

For the queries the config should only be used to extract the rpc address. We should be able to query connections that are not in the relayer configuration.

Reply:
Added separate issue #204 for this